.gitignore

```diff
@@ -270,14 +270,17 @@ BenchmarkDotNet.Artifacts/
 # VSTest results
 *.trx
 
-deploy/linux/
-deploy/AzureAppServices/
-!build/artifacts
-
 # macOS temporal files
 *..DS_Store
 .DS_Store/*
 .DS_Store
 
-# profiler output files
-src/Datadog.Trace.ClrProfiler.Native/build/
\ No newline at end of file
+## Tracer .gitignore settings
+# Tracer deployment artifacts
+tracer/deploy/linux/
+tracer/deploy/AzureAppServices/
+!tracer/build/artifacts
```

Comment:
`!tracer/build/artifacts` seems correct, because there's a file in that we don't want to ignore. I'm guessing this was added because this line ignores folders named `artifacts`. Maybe the better fix is to remove both.
https://github.com/DataDog/dd-trace-dotnet/blob/13cdc48e4f70541c07256e574f7d6ae7efeb4ab8/.gitignore#L49

Reply:
Regarding `deploy/`, the wix project that builds the msi used to live there before we moved it in #973. I don't think we use `deploy/AzureAppServices/`, either. @colin-higgins may know better.